### PR TITLE
[MODULAR] Reworks and changes up three donator items

### PR DESCRIPTION
**Pull request by zydras**
## About The Pull Request

This PR does a few things, specifically:
- Adds a proper unique sprite to the FAR-14C donator item (farsightednightlight's)
- Removes one of my donator items (Korpstech Genetics Scanner)
- Adds a new donator item in its place (Engine Technician Harness)

Screenshot (1565) Screenshot (1564) Screenshot (1563)
(Y'gotta respect the drip)
Also adds a friend's donator item I sprited but didn't add it to code yet, displayed below in a reply.

## How This Contributes To The Skyrat Roleplay Experience

It makes an item unique and it makes another one, well, actually be used. I stopped using the donator genetics scanner, so y'know.
## Changelog

:cl:
add: FAR-14C donator item now has its own unique sprite.
add: Tactichill Jacket donator item.
del: Removed an unused donator item (Korpstech Genetics Scanner).
add: A replacement for said unused donator item (Engine Technician Harness).
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Is this harness meant to be for a syndicate role? I dont feel good about openly-syndicate gear on station, especially when its non-antag gear
The sprite is dope tho

**Comment by zydras** [comment]
> Is this harness meant to be for a syndicate role? I dont feel good about openly-syndicate gear on station, especially when its non-antag gear The sprite is dope tho

Oh yes it 100% is meant to be for DS-2, but since it's a ghostrole and not a job by itself I don't think there's a tag for it to be locked _outside_ from station jobs. But hey, i'm a roleplay mentor of course i'm not going to be bringing it on-station. No, it's meant solely for DS-2 for my syndicate character, and no other.

**Comment by ghost** [comment]
> > Is this harness meant to be for a syndicate role? I dont feel good about openly-syndicate gear on station, especially when its non-antag gear The sprite is dope tho
> 
> Oh yes it 100% is meant to be for DS-2, but since it's a ghostrole and not a job by itself I don't think there's a tag for it to be locked _outside_ from station jobs. But hey, i'm a roleplay mentor of course i'm not going to be bringing it on-station. No, it's meant solely for DS-2 for my syndicate character, and no other.

Alright, just wanted to make sure. If it was meant for the station.. yeesh.
Either way we (code as a whole not you specifically) shouldddd look into offstation-locked items, for cases just like this one. That or it could be a reskin kit for an item you spawn with only off-station; I know guns were meant to have donor reskin kits so its not too out there for a vest

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -1224,16 +1222,12 @@
...
-	icon_state = "korpsgenetic"
-	inhand_icon_state = "korpsgenetic"
-	worn_icon_state = "korpsgenetic"
-	icon = 'modular_skyrat/master_files/icons/donator/obj/custom.dmi'
-	lefthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_left.dmi'
-	righthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_right.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/custom_w.dmi'
+/obj/item/clothing/suit/jacket/gorlexharness
+	name = "engine technician harness"
+	desc = "A blood-red engineering technician harness. You can't seem to figure out an use to it, but it seems it's magnetized at places."
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
	desc = "A blood-red engineering technician harness. You can't seem to figure out a use to it, but it seems to seal magnetically in some places."
```
`A use`, not `An use` ;)
Second part I might have misunderstood, I assume it's meaning magnetism as a function and not a property - if the latter, `but the plating seems magnetic in some places` would do as well

Comment (ghost) [reply modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: Other than this small grammar note, it all checks out, sprites are badass too 👍

Comment (zydras) [reply modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: Oh thanks, I almost missed out on those two little mistakes, ahaha. And, thanks!

**Comment by zydras** [comment]
> Alright, just wanted to make sure. If it was meant for the station.. yeesh. Either way we (code as a whole not you specifically) shouldddd look into offstation-locked items, for cases just like this one. That or it could be a reskin kit for an item you spawn with only off-station; I know guns were meant to have donor reskin kits so its not too out there for a vest

In a near future, that indeed would definitely be nice

**Comment by zydras** [comment]
Someone asked to have their donator item properly added to the game since they already had the sprite, this should do it.
(Unityhominem/Kay-Nite)
Screenshot (1567)

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 2]
```diff
@@ -1224,16 +1222,12 @@
...
-	name = "Korpstech genetics scanner"
-	desc = "A hand-held sequence scanner for analyzing someone's gene sequence on the fly. This one is bright pink and has some kind of Helix shape on its back."
-	icon_state = "korpsgenetic"
-	inhand_icon_state = "korpsgenetic"
-	worn_icon_state = "korpsgenetic"
-	icon = 'modular_skyrat/master_files/icons/donator/obj/custom.dmi'
-	lefthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_left.dmi'
-	righthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_right.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/custom_w.dmi'
+/obj/item/clothing/suit/jacket/gorlexharness
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
/obj/item/clothing/suit/jacket/gorlex_harness
```

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 3]
```diff
@@ -363,11 +363,10 @@
 	item_path = /obj/item/card/fuzzy_license
 	ckeywhitelist = list("fuzlet")
 
-/datum/loadout_item/pocket_items/korpstech_scanner
-	name = "Korpstech Genetics Scanner"
-	item_path = /obj/item/sequence_scanner/korpstech
+/datum/loadout_item/suit/gorlexharness
+	name = "Engine Technician Harness"
+	item_path = /obj/item/clothing/suit/jacket/gorlexharness
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
/datum/loadout_item/suit/gorlex_harness
	name = "Engine Technician Harness"
	item_path = /obj/item/clothing/suit/jacket/gorlex_harness
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by GoldenAlpharex — APPROVED** [review]
I verified the other donator's eligibility, they're clear to go.

**Pull request merged**